### PR TITLE
CASMINST-4798 Document UAN Minimized Downtime to main

### PR DESCRIPTION
**Pull request by trad511**
# Description

Added overview of the procedure and timelines where running UAN can be upgraded with minimal downtime.  Also changed the SLS Upgrader examples by removing sample VLAN and subnet information to prevent cut-and-paste misconfiguration of systems.

# Checklist Before Merging

- Requires CASMUSER-3024 (https://github.com/Cray-HPE/docs-csm/pull/1853)

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/MTL-1695/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/network/management_network/bican_enable.md [hunk 1]
```diff
@@ -20,33 +21,70 @@
...
 `uan_can_setup: yes`
 
 Please refer to the "HPE Cray User Access Node (UAN) Software Administration Guide (`S-8033`)" document on the [HPE Support Center](https://support.hpe.com) website for more information.
 
+### Minimize UAN Downtime
+
+UAN running before and during an upgrade to CSM 1.2 will continue running with no connectivity or local data impacts until an administrator-scheduled transition takes place.
+While access to currently running UANs continues during the upgrade, UAN rebuilds and reboots not supported.
+
+The time frame over which the transition can be scheduled is quite large and the transition requires only that UAN users to log out of the UAN (over the old IPv4 address) and log back in (over a new IPv4 address).
```
Comment (nrockershousen) [inline operations/network/management_network/bican_enable.md:68]:
```suggestion
The time frame over which the transition can be scheduled is quite large and the transition requires only that UAN users log out of the UAN (over the old IPv4 address) and log back in (over a new IPv4 address).
```

File: operations/network/management_network/bican_enable.md [hunk 2]
```diff
@@ -20,33 +21,70 @@
...
+The following diagram illustrates the UAN timeline before, during and after the CSM 1.2 upgrade.
```
Comment (nrockershousen) [inline operations/network/management_network/bican_enable.md:69]:
```suggestion
The following diagram illustrates the UAN timeline before, during, and after the CSM 1.2 upgrade.
```

File: operations/network/management_network/bican_enable.md [hunk 3]
```diff
@@ -20,33 +21,70 @@
...
+
+![UAN Upgrade Transitions](../../../img/UAN_transition_CSM_1.2.png)
+
+Concretely, users on running UAN may transitioned from the CMN to the new CAN between the two following upgrade points:
```
Comment (nrockershousen) [inline operations/network/management_network/bican_enable.md:73]:
```suggestion
Concretely, users on a running UAN may be transitioned from the CMN to the new CAN between the two following upgrade points:
```

File: operations/network/management_network/bican_enable.md [hunk 4]
```diff
@@ -20,33 +21,70 @@
...
+
+1. After [SLS Upgrade](../../../upgrade/1.2/Stage_0_Prerequisites.md#stage-03---upgrade-management-network) has been completed, but
```
Comment (nrockershousen) [inline operations/network/management_network/bican_enable.md:75]:
```suggestion
1. After [SLS Upgrade](../../../upgrade/1.2/Stage_0_Prerequisites.md#stage-03---upgrade-management-network) has been completed.
```

File: upgrade/1.2/Stage_0_Prerequisites.md [hunk 5]
```diff
@@ -119,68 +128,78 @@ CSM 1.2 introduces the bifurcated CAN (BICAN) as well as network configuration c
...
+1. Extract SLS data to a file.
 
    ```bash
-   curl -k -H "Authorization: Bearer ${TOKEN}" https://api-gw-service-nmn.local/apis/sls/v1/dumpstate | jq -S . > sls_input_file.json
+   ncn-m001# curl -k -H "Authorization: Bearer ${TOKEN}" https://api-gw-service-nmn.local/apis/sls/v1/dumpstate | jq -S . > sls_input_file.json
    ```
 
 ### Migrate SLS data JSON to CSM 1.2
 
+<<<<<<< HEAD
```
Comment (nrockershousen) [inline upgrade/1.2/Stage_0_Prerequisites.md:165]: This conflict marker should be removed. Looks like lines 165-167 can be deleted.

File: upgrade/1.2/Stage_0_Prerequisites.md [hunk 6]
```diff
@@ -236,29 +254,29 @@ curl --fail -H "Authorization: Bearer ${TOKEN}" -k -L -X POST 'https://api-gw-se
...
-1. (`ncn-m001#`) Unset the `NEXUS_PASSWORD` variable, if it was set in the earlier step.
+1. Unset the `NEXUS_PASSWORD` variable, if it was set in the earlier step.
 
    ```bash
-   unset NEXUS_PASSWORD
+   ncn-m001# unset NEXUS_PASSWORD
    ```
 
-1. (`ncn-m001#`) Commit changes to `customizations.yaml` (optional).
+1. Commit changes to `customizations.yaml` (optional).
```
Comment (nrockershousen) [inline upgrade/1.2/Stage_0_Prerequisites.md:279]:
```suggestion
1. (Optional) Commit changes to `customizations.yaml`.
```

**Review by nrockershousen — CHANGES_REQUESTED** [review]

File: operations/network/management_network/bican_enable.md [hunk 7]
```diff
@@ -20,33 +21,70 @@
...
+```
 
-Run the following command to update SLS with `CHN` as the `SystemDefaultRoute`
+**Preview:** High Speed Network access by users to UAN, UAI and API resources is the `CHN` option.
+This is available during the CSM 1.2 release for those who wish to forge ahead of release-supported features.
+To set and use the `CHN` in SLS update the `SystemDefaultRoute` with:
 
 ```bash
-/usr/share/doc/csm/scripts/operations/bifurcated_can/bican_route.py --route CHN
+ncn# /usr/share/doc/csm/scripts/operations/bifurcated_can/bican_route.py --route CHN
```
Comment (nrockershousen) [inline operations/network/management_network/bican_enable.md:49]: Ensure command prompts are following the new convention:

https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#command-prompt-conventions

Comment (mharding-hpe) [reply operations/network/management_network/bican_enable.md:49]: > Ensure command prompts are following the new convention:
> 
> https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#command-prompt-conventions

For main branch, we do need to include the command prompt somewhere to indicate where the command is to be run, but it doesn't have to be inside the fenced block itself. This was the change that allowed easier copy and paste. I do think this should probably be more explicitly clarified in the conventions page, though.

**Review by mharding-hpe — APPROVED** [review]

**Comment by mharding-hpe** [comment]
I did some linting.

**Review by rustydb — APPROVED** [review]

**Pull request merged**